_data/banking.yml

```diff
@@ -64,6 +64,13 @@ websites:
       - email
     doc: https://www.americaschristiancu.com/ministry/services/online-banking-bill-pay.html
 
+  - name: Ameriprise Financial
+    url: https://www.ameriprise.com
+    twitter: ameriprise
+    facebook: ameriprise
+    img: ameriprise.png
+    email_address: anti.fraud@ampf.com
```

Comment:
I don't think the Anti Fraud department is the correct place to contact for this.
```suggestion
```